package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "dev",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "mocha test"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/reactivepixel/DynamicFormGen.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/reactivepixel/DynamicFormGen/issues"
+  },
+  "homepage": "https://github.com/reactivepixel/DynamicFormGen#readme",
+  "devDependencies": {
+    "chai": "^4.1.2",
+    "eslint-config-prettier": "^2.9.0",
+    "mocha-eslint": "^4.1.0",
+    "prettier": "1.11.0",
+    "eslint-config-airbnb": "^16.1.0",
```

Comment:
This `eslint-config-airbnb` could be removed since the airbnb-base version is being used. I would also recommend adding anything related to eslint as a devDependency since they would not necessarily be needed to run the program.

Reply:
Everything else looks good though!